core/network/impl/protocols/protocol_base_impl.hpp

```diff
@@ -96,6 +104,7 @@ namespace kagome::network {
    private:
     Host &host_;
     Protocol const protocol_;
```

Comment:
Can't understand if this field is still needed and how it is initialized, please remove if it is not needed

Reply:
Join to question.